### ISSUE TITLE
.cue sheet support

### ISSUE THREAD

**Issue by intranauta** [jfchapman/VUPlayer#31]
Hi James,
Great v.4.14 release, thanks a lot for your work! VUPlayer is my one and only audio player since 2007, and will continue to be :)
I have a request, for a future realease: I'll be possible add back cue sheet support? That already existed in v.2.4x but not in v4, and would help for FLAC, APE or WV one-archive rips with a .cue sheet file with the tracks info.
Thanks & regards
Emilio.

**Comment by jfchapman** [comment]
Hi, it's good to hear from a long time user.
Cue sheet support is one of the things I never got round to adding to the new version, but it's on the list of things to do.
I'll probably have to limit support to the playlists, as I don't see a nice & easy way of adding .cue file support to the folder browser.

Thanks,
James

**Comment by jfchapman** [comment]
Hi,

It took awhile, but I finally added support for cue sheets in the latest 4.18 release.
Specifically, the 'Import Playlist' menu option now allows you to import .cue files.
I also included support for retrieving metadata from the MusicBrainz service if the cue sheet is a suitable type, i.e. it has a single source file, and continuous cues, with a start cue of zero. These are the only type of cue sheets I had available for testing, so hopefully it's the most common type.

Thanks,
James
